test/schema.sql

```diff
@@ -148,13 +148,14 @@ CREATE TABLE users (
 
 CREATE TABLE statements (
   id VARCHAR(66) NOT NULL,
-  ipfs VARCHAR(64) NOT NULL,
+  ipfs VARCHAR(64) DEFAULT NULL,
   delegate VARCHAR(100) NOT NULL,
   space VARCHAR(100) NOT NULL,
   about TEXT,
   statement TEXT,
   network VARCHAR(24) NOT NULL DEFAULT 's',
   discourse VARCHAR(64),
+  source VARCHAR(100) DEFAULT NULL,
```

Comment:
maybe we should default to 'snapshot' or something?

Reply:
>  If the user edit his statement on Snapshot we would remove the source from the table.

It was in the pitch spec to keep it empty